src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -128,7 +172,7 @@ public double getGyroVelocity() {
   @Override
   public void simulationPeriodic() {
     double degreesPerSecond = Units.radiansToDegrees(getRobotRelativeSpeeds().omegaRadiansPerSecond);
-    simGyroAngle.set(simGyroAngle.get() - (degreesPerSecond * 0.02));
+    simGyroAngle.set(simGyroAngle.get() - (degreesPerSecond * DEFAULT_DT));
```

Comment:
DT isn't configurable so it's not reaaally a default. it's hardcoded to 20ms in TimedRobot

Reply:
okay I will keep that then